src/nextjournal/clerk/viewer.cljc

```diff
@@ -16,7 +16,10 @@
                    (javax.imageio ImageIO)
                    (java.util Base64))))
 
-(defrecord ViewerEval [form])
+(defrecord ViewerEval [form]
+  #?@(:cljs [IFn
+             (-invoke [_this x] ((eval form) x))
+             (-invoke [_this x y] ((eval form) x y))]))
```

Comment:
This is sub-optimal but allows to treat `ViewerEval` instances as valid hiccup/reagent tags. It's needed when calling `viewer/describe` (and hence `viewer/inspect-leafs`) in a cljs context. Maybe we can find a more explicit approach or rename `ViewerEval` which so far should only be called with `v/inspect` as its form.

If we don't implement `IFn` we'll get: 

```
Error
Assert failed: Invalid Hiccup form: [#nextjournal.clerk.viewer.ViewerEval{:form v/inspect} {:path [], :nextjournal/value ["h3" {:id "Hello%20Markdown"} [#nextjournal.clerk.viewer.ViewerEval{:form v/inspect} {:path [], :nextjournal/value [:span "Hello Markdown"], :nextjournal/viewer {:name :html, :render-fn #nextjournal.clerk.viewer.ViewerFn{:form v/html, :f nextjournal.clerk.sci_viewer.html_viewer}}}]], :nextjournal/viewer {:name :html, :render-fn #nextjournal.clerk.viewer.ViewerFn{:form v/html, :f nextjournal.clerk.sci_viewer.html_viewer}}}]
 (in nextjournal.clerk.sci_viewer.inspect)
(valid-tag? tag)
```

Note that we need to prepend a describe to all wrapped values with viewers which don't have a render-fn. This is the case of the markdown viewer: we'll need expressions like

 ```clojure
[inspect (viewer/describe (md "# Markdown"))]
```

for it to render correctly.